packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Cover.Example.tsx

```diff
@@ -24,6 +32,63 @@ const DIRECTION_OPTIONS = [
   { key: DirectionalHint.rightBottomEdge, text: 'Right Bottom Edge' }
 ];
 
+const theme = getTheme();
+const styles = mergeStyleSets({
+  buttonArea: {
+    verticalAlign: 'top',
+    display: 'inline-block',
+    textAlign: 'center'
+  },
+  configArea: {
+    minWidth: '300px',
+    display: 'inline-block'
+  },
+  callout: {
+    maxWidth: 300
+  },
+  calloutExampleButton: {
+    width: '100%'
+  },
+  header: {
+    padding: '18px 24px 12px'
+  },
+  title: [
+    theme.fonts.xLarge,
+    {
+      margin: 0,
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.semilight
+    }
+  ],
+  inner: {
+    height: '100%',
+    padding: '0 24px 20px'
+  },
+  actions: {
```

Comment:
Not seeing this used in any of the examples

Reply:
Removed!